src/plugins/creditor/declaration.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType, EdgeType} from "../../analysis/types";
+import {NodeAddress, EdgeAddress} from "../../core/graph";
+
+export const nodePrefix = NodeAddress.fromParts(["sourcecred", "creditor"]);
+export const edgePrefix = EdgeAddress.fromParts(["sourcecred", "creditor"]);
+
+export const CreditorEntryType: NodeType = deepFreeze({
+  name: "CredNode",
```

Comment:
IMO calling it a "CredNode" is more confusing either than "Nodes" or "Creditor Nodes". 
I vote either "Creditor Node" or "Manual Node"